plugins/producer_plugin/test/test_snapshot_scheduler.cpp

```diff
@@ -19,8 +19,8 @@ BOOST_AUTO_TEST_CASE(snapshot_scheduler_test) {
 
    {
       // add/remove test
-      producer_plugin::snapshot_request_information sri1 = {.block_spacing = 100, .start_block_num = 5000, .end_block_num = 10000, .snapshot_description = "Example of recurring snapshot"};
-      producer_plugin::snapshot_request_information sri2 = {.block_spacing = 0, .start_block_num = 5200, .end_block_num = 5200, .snapshot_description = "Example of one-time snapshot"};
+      snapshot_scheduler::snapshot_request_information sri1 = {.block_spacing = 100, .start_block_num = 5000, .end_block_num = 10000, .snapshot_description = "Example of recurring snapshot"};
```

Comment:
```suggestion
      using snapshot_request_information = snapshot_scheduler::snapshot_request_information;
      snapshot_request_information sri1 = {.block_spacing = 100, .start_block_num = 5000, .end_block_num = 10000, .snapshot_description = "Example of recurring snapshot"};
```

and remove following `snapshot_scheduler::`